antenna-documentation/src/site/markdown/analyzers/csv-analyzer-step.md.vm

```diff
@@ -0,0 +1,51 @@
+## CSV analyzer
+This analyzer investigates a given *csv* file. The format of this csv file follows this format:
+
+```
+"Artifact Id","Group Id","Version","License Short Name","License Long Name","File Name","Coordinate Type","Observed License","Source URL","Release Tag URL","Software Heritage URL","Clearing State","Change Status","CPE","Copyrights"
```

Comment:
I struggle a bit here. I think we should go through the list and check it, because it should at least store all information that storable in SW360 if we want to use this as basis for curations by Sheriff.